utils/reviewcomment.go

```diff
@@ -26,15 +26,74 @@ const (
 	SastComment       ReviewCommentType = "Sast"
 )
 
-func AddReviewComments(repo *Repository, pullRequestID int, client vcsclient.VcsClient, issues *IssuesCollection) (err error) {
-	if err = deleteOldReviewComments(repo, pullRequestID, client); err != nil {
-		err = errors.New("couldn't delete pull request review comment: " + err.Error())
+func HandlePullRequestCommentsAfterScan(issues *IssuesCollection, repo *Repository, client vcsclient.VcsClient, pullRequestID int) (err error) {
+	if !repo.Params.KeepPreviousComments {
+		log.Debug("Looking for an existing Frogbot pull request comments. Deleting them if exists...")
+		if err = DeleteExistingPullRequestComments(repo, client); err != nil {
+			err = errors.New("couldn't delete pull request comment: " + err.Error())
+			return
+		}
+		if err = deleteExistingPullRequestReviewComments(repo, pullRequestID, client); err != nil {
+			err = errors.New("couldn't delete pull request review comment: " + err.Error())
+			return
+		}
```

Comment:
The above code section includes two deletion function calls - 
1. DeleteExistingPullRequestComments
2. deleteExistingPullRequestReviewComments

It is unclear for the reader what is the diferemce between these two calls. Let's clerify this using a comment here. Let's also add dewcription comments on thsese two functions.